static/app/components/core/avatar/index.stories.tsx

```diff
@@ -0,0 +1,173 @@
+import {Fragment} from 'react';
+
+import {DocIntegrationAvatar} from 'sentry/components/core/avatar/docIntegrationAvatar';
+import {OrganizationAvatar} from 'sentry/components/core/avatar/organizationAvatar';
+import {ProjectAvatar} from 'sentry/components/core/avatar/projectAvatar';
+import {SentryAppAvatar} from 'sentry/components/core/avatar/sentryAppAvatar';
+import {TeamAvatar} from 'sentry/components/core/avatar/teamAvatar';
+import {UserAvatar} from 'sentry/components/core/avatar/userAvatar';
```

Comment:
Should we have the stories use the base `<Avatar />` component for consistency?

Reply:
yeah, I will followup and do that. I would like to fixup the SentryApp and project components before I do that!